lib/td/command/export.rb

```diff
@@ -32,6 +32,13 @@ def export_result(op)
       retry_limit: retry_limit,
       priority: priority,
     }
+    if wait
+      job = client.job(target_job_id)
+      if !job.finished?
+        print "target job is still "
```

Comment:
"still running" or something like that, or `wait_job` print some text continued from this?